src/Sylius/Bundle/AdminBundle/Templating/Helper/AvailableLocaleHelper.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\AdminBundle\Templating\Helper;
+
+use Sylius\Component\Resource\Translation\Provider\TranslationLocaleProviderInterface;
+use Symfony\Component\Templating\Helper\Helper;
+
+class AvailableLocaleHelper extends Helper
```

Comment:
I think we could get rid of those `helpers` and move the implementation of those directly into Twig extensions - we don't use nor support any other templating system.